Mixpanel/MPNotification.h

```diff
@@ -1,6 +1,12 @@
 #import <Foundation/Foundation.h>
+#if !TARGET_OS_WATCH
+#import <JavaScriptCore/JavaScriptCore.h>
+#endif
 #import "MPDisplayTrigger.h"
 
+extern NSString* const PROPERTY_EVAL_FUNC_NAME;
```

Comment:
do you have to make `PROPERTY_EVAL_FUNC_NAME` and `PROPERTY_FILTERS_JS_URL` public?
You can just `static NSString *const kConstant = @"myconstant";` in MPNotification.m
kConstant is the naming convention for private constant in ObjC.